docs/user.md

```diff
@@ -129,4 +129,7 @@ companyItemImg {
     <div class="companyItem">
        <img class="companyItemImg" src="../assets/logo/users/cisco.png">
     </div>
+    <div class="companyItem">
+       <img class="companyItemImg" src="../assets/logo/users/netease_game.png">
```

Comment:
The logo picture needs to use png file format?

Reply:
fixed